### DISCUSSION TITLE
101

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#13] [General]
# 潮流周刊第101期 - 1024节日快乐

封面来源于周六径山花海的航拍图，风和日丽露营还是很舒服的。今天刚好是 1024 程序员节日，一直觉得日子过得还是需要有些仪式感才比较好，周末写了一个小彩蛋「

https://weekly.tw93.fun/posts/101-1024%E8%8A%82%E6%97%A5%E5%BF%AB%E4%B9%90

**Comment by chang8128** [comment]
不错哦

**Comment by yaokaiqd** [comment]
博主你好,很喜欢「祝你 1024 节日快乐」这个效果,想了解下是如何实现(在页面内实时展示效果,及代码的动效)的？

**Reply by tw93** [reply]
可以去这里看看，我提交到掘金了
https://code.juejin.cn/pen/7160299226851704839
